server/src/meguca/assets/common.go

```diff
@@ -45,3 +66,83 @@ func (s *FileStore) Get(board string) (file File) {
 	}
 	return file
 }
+
+func generateVideoNames() {
```

Comment:
Needs error handling.

Reply:
Oh, there is.

Reply:
Okay, we can avoid recompilation if `err != nil`.

Reply:
Wait, no, I want to keep recompilation, because if the directory is empty or doesn't exist, the client should show [none].